docs/overview.rst

```diff
@@ -294,6 +294,53 @@ Cryptographic Hashing
         >>> Web3.solidityKeccak(['address'], ["ethereumfoundation.eth"])
         HexBytes("0x913c99ea930c78868f1535d34cd705ab85929b2eaaf70fcd09677ecd6e5d75e9")
 
+.. py:classmethod:: Web3.sha3(primitive=None, hexstr=None, text=None)
```

Comment:
I added these back in because I noticed they haven't gone through a deprecation cycle. They were deprecated in v5a1.